abdulfatai360/banka
# #165954816 Implement feature for client to view and create account

#### What does this PR do?
Add feature for user to view his accounts details and open a new account

#### Description of Task to be completed?
- Have required API endpoints integrated to the client's `accounts` page
- Have user redirected to the `signin` page if he has not signed in
- Enable client user to view his account details
- Enable client user to open a new account
- Validate the user inputs when opening a new account
- Return appropriate error messages where required 

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/65/head:ft-client-view-create-account-165954816`
- Now that you have a copy of the branch, switch to it using  `git checkout ft-client-view-create-account-165954816`
- Your directory will now be an exact copy of the PR

#### What are the relevant pivotal tracker stories?
[#165954816](https://www.pivotaltracker.com/story/show/165954816)
